File: PMS/Forms/MainForm.vb
Public Partial Class MainForm
    Private Sub LoginButton_Click(sender As Object, e As EventArgs) Handles LoginButton.Click
        If UsernameTextBox.Text = "Admin" AndAlso PasswordTextBox.Text = "Password" Then
            OperationsGroupBox.Enabled = True
            CredentialGroupBox.Enabled = False
        Else
            MessageBox.Show("The username/password is incorrect.", "Invalid Credentials", MessageBoxButtons.OK, MessageBoxIcon.[Error])
        End If
    End Sub

    Private Sub TextBox_TextChanged(sender As Object, e As EventArgs) Handles UsernameTextBox.TextChanged, PasswordTextBox.TextChanged
        LoginButton.Enabled = Not (UsernameTextBox.Text = String.Empty OrElse PasswordTextBox.Text = String.Empty)
    End Sub

    Private Sub OpenButton_Click(sender As Object, e As EventArgs) Handles OpenButton.Click
        Dim selectedOperation As Object = OperationsGroupBox.Controls.OfType(Of RadioButton)().FirstOrDefault(Function(r) r.Checked)
        Select Case selectedOperation.Name
            Case "ProjectRegistrationRadioButton"
                ProjectRegistration.Show()
                Exit Select

            Case "ProgressUpdateRadioButton"
                UpdateProgress.Show()
                Exit Select

            Case "GenerateTimesheetRadioButton"
                GenerateTimesheet.Show()
                Exit Select

            Case "GenerateBillRadioButton"
                GenerateBill.Show()
                Exit Select
        End Select
    End Sub

    Private Sub AboutStatusStripLabel_Click(sender As Object, e As EventArgs) Handles AboutStatusStripLabel.Click
        About.Show()
    End Sub
End Class

File: PMS/Forms/MainForm.cs
using System;
using System.Linq;
using System.Windows.Forms;

namespace PMS
{
    public partial class MainForm : Form
    {
        public MainForm()
        {
            InitializeComponent();
        }

        private void LoginButton_Click(object sender, EventArgs e)
        {
            if (UsernameTextBox.Text == "Admin" && PasswordTextBox.Text == "Password")
            {
                OperationsGroupBox.Enabled = true;
                CredentialGroupBox.Enabled = false;
            }
            else
            {
                MessageBox.Show("The username/password is incorrect.", "Invalid Credentials", MessageBoxButtons.OK,
                    MessageBoxIcon.Error);
            }
        }

        private void TextBox_TextChanged(object sender, EventArgs e)
        {
            LoginButton.Enabled = !(UsernameTextBox.Text == string.Empty || PasswordTextBox.Text == string.Empty);
        }

        private void OpenButton_Click(object sender, EventArgs e)
        {
            var selectedOperation = OperationsGroupBox.Controls.OfType<RadioButton>()
                .FirstOrDefault(r => r.Checked);
            switch (selectedOperation.Name)
            {
                case "ProjectRegistrationRadioButton":
                    new ProjectRegistration().Show();
                    break;

                case "ProgressUpdateRadioButton":
                    new UpdateProgress().Show();
                    break;

                case "GenerateTimesheetRadioButton":
                    new GenerateTimesheet().Show();
                    break;

                case "GenerateBillRadioButton":
                    new GenerateBill().Show();
                    break;
            }
        }

        private void AboutStatusStripLabel_Click(object sender, EventArgs e)
        {
            new About().Show();
        }
    }
}